src/SamplesApp/SamplesApp.Shared/App.xaml.cs

```diff
@@ -95,6 +95,36 @@ override void OnLaunched(LaunchActivatedEventArgs e)
 				// this.DebugSettings.EnableFrameRateCounter = true;
 			}
 #endif
+			InitializeFrame(e.Arguments);
+			Windows.UI.Xaml.Window.Current.Activate();
+
+			DisplayLaunchArguments(e);
+		}
+
+		protected
+#if HAS_UNO
+			internal
+#endif
+			override async void OnActivated(IActivatedEventArgs e)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Return 'Task' instead.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5378714)

Reply:
not applicable